### ISSUE TITLE
Game Bots

### ISSUE THREAD

**Issue by THB0705** [Ben-Ryder/Conqueror-of-Empires#2]
Will you be working to implement an game bot to play against players who have no friends to play with ;) ?

**Comment by ben-ryder** [comment]
It was an idea I had yes :)
I do have a few other bits I want to polish and work on, mainly tuning the mechanics as it gets a bit one-sided late game, but also adding a bit more depth to the games strategy.
Hopefully I will get to developing bots soon-ish. I definitely can't promise a human skill player straight away! But definitely one you can play against.

**Comment by THB0705** [comment]
Yes, that's good. For me its hard to play-test the game as playing against yourself is quite dull. So even a dumb bot, would be a great addition, to begin with.

**Comment by ben-ryder** [comment]
This is now being worked on in the [GH-2_computer-players](https://github.com/Ben-Ryder/Conqueror-of-Empires/tree/GH-2_computer-players) branch.
